.github/appveyor.yml

```diff
@@ -0,0 +1,28 @@
+version: 3.3.6+.{build}
```

Comment:
As noted in the comments above, let's not try to deal with appveyor.  There should only be a handful of further commits to the 3.3 branch before it is EOLed.  So delete this file.